narwhal/types/src/primary.rs

```diff
@@ -385,6 +421,9 @@ pub enum SystemMessage {
     DkgConfirmation(
         fastcrypto_tbls::dkg::Confirmation<<ThresholdBls12381MinSig as ThresholdBls>::Public>,
     ),
+    // `RandomnessSignature` contains random bytes generated by the random beacon protocol
+    // for the given round. Can be verified using the VSS public key derived from DKG.
+    RandomnessSignature(RandomnessRound, RandomnessSignature),
```

Comment:
I just remembered that we should store relatively complex fastcrypto types as bytes in Header, similar to `AggregateSignatureBytes` in Narwhal and `AuthorityPublicKeyBytes` in Sui. The advantages are:
- Avoiding serialization and deserialization cost since these fields are only used in a few places.
- Make updating `narwhal/node/tests/staged/narwhal.yaml` and `narwhal-node test_format` easier.

I think all three variants in SystemMessage should be bytes instead. cc @benr-ml

Reply:
https://github.com/MystenLabs/sui/pull/14706